api/views/phones.py

```diff
@@ -592,21 +619,8 @@ def inbound_sms_iq(request: Request) -> response.Response:
     if inbound_contact:
         _check_and_update_contact(inbound_contact, "texts", relay_number)
 
-    iq_formatted_real_num = real_phone.number.replace("+", "")
-    iq_formatted_relay_num = relay_number.number.replace("+", "")
     text = message_body(inbound_from, inbound_body)
-    json_body = {
-        "from": iq_formatted_relay_num,
-        "to": [iq_formatted_real_num],
-        "text": text,
-    }
-    resp = requests.post(
-        "https://messagebroker.inteliquent.com/msgbroker/rest/publishMessages",
-        headers={"Authorization": f"Bearer {settings.IQ_OUTBOUND_API_KEY}"},
-        json=json_body,
-    )
-    if not resp.status_code == 200:
-        raise exceptions.ValidationError("Call data missing Caller or Called.")
+    _send_iq_sms(relay_number.number, real_phone.number, text)
 
     incr_if_enabled("phones_outbound_sms_iq")
```

Comment:
suggestion (non-blocking): add a `phones_outbound_sms_iq` increment at `_send_iq_sms` call to measure how many text messages were sent using iQ.

Reply:
added.